src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -85,10 +84,10 @@ def convert_iso8601_to_datetime(
 
 
 class EnsembleSnapshotMetadata(TypedDict):
-    # contains the QColor used in the GUI for each fm_step
-    aggr_fm_step_status_colors: defaultdict[RealId, dict[FmStepId, QColor]]
-    # contains the QColor used in the GUI for each real
-    real_status_colors: dict[RealId, QColor]
+    # contains state mapped to QColor used in the GUI for each fm_step
```

Comment:
Not sure if these should be removed as well.

Reply:
The comment? Maybe it does not add much value no. It kind of still says that this dict is the map to QColor, but it is only meant to be used for that (?)